packages/theming/apple-theme/src/appleTheme.ts

```diff
@@ -0,0 +1,30 @@
+import { Spacing, Theme } from '@fluentui-react-native/theme-types';
+import { getFluentUIApplePalette } from './fluentAppleColors';
+import { getAppleSemanticPalette } from './applePlatformColors';
+
+import { paletteFromAppleColors } from './appleTheme.colors';
+
+import { appleTypography } from './appleTheme.typography';
+
+export function appleSpacing(): Spacing {
+  return { s2: '4px', s1: '8px', m: '16px', l1: '20px', l2: '32px' };
+}
+
+export const appleComponents = {
+  Button: {
+    tokens: {
```

Comment:
Where do these tokens come from? Wouldn't it vary between iOS/macOS?

Reply:
They would. A few of these pages were initially split out between macOS and iOS before I reverted the iOS changes and just had the macOS changes. As such, all the code you see now is intended for macOS and will fail on iOS

```
export function createAppleTheme(): ThemeReference {
  const appleTheme = Platform.select({
    macos: () => new ThemeReference(defaultAppleThemeMacOS),
    ios: () => {
      console.log('iOS Theme not currently supported, using default theme');
      return new ThemeReference(createDefaultTheme());
    },
  })();
  return appleTheme;
 ```

Reply:
You could define these in separate files (i.e. theme.macos.ts and theme.ios.ts) if it makes more sense.  I don't know how much cross over there will be

Reply:
I'll leave that as a task for when we add the iOS theme.
The tokens here came from looking at the design of the [FluentUI Apple macOS button](https://github.com/microsoft/fluentui-apple/blob/4b9992b3bbeb46776de86d0c8c14ac3c08a207f0/macos/FluentUI/Button/Button.swift#L584)

Reply:
Decided to split the rest of the fies by platform, even if we just have a macOS theme.